nailgun/entities.py

```diff
@@ -4757,6 +4757,19 @@ def __init__(self, server_config=None, **kwargs):
         self._meta = {'api_path': 'api/v2/http_proxies', 'server_modes': ('sat')}
         super(HTTPProxy, self).__init__(server_config, **kwargs)
 
+    def update_payload(self, fields=None):
+        """Wrap submitted data within an extra dict."""
+        return {u'http_proxy': super(HTTPProxy, self).update_payload(fields)}
```

Comment:
There's no need for `u` now that #702 is merged.

Reply:
removed all the explicit unicode (`u`)